ci-operator/step-registry/upi/conf/vsphere/upi-conf-vsphere-commands.sh

```diff
@@ -20,7 +20,8 @@ openshift_install_path="/var/lib/openshift-install"
 third_octet=$(grep -oP '[ci|qe\-discon]-segment-\K[[:digit:]]+' <(echo "${LEASED_RESOURCE}"))
 
 export HOME=/tmp
-export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=${RELEASE_IMAGE_LATEST}
+#export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=${RELEASE_IMAGE_LATEST}
```

Comment:
ah, there was a line to set `OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE`, if that I would withdraw my last comment in https://github.com/openshift/release/pull/40211#discussion_r1224074058, maybe we can add `RELEASE_IMAGE_LATEST` dependency back to help user easy-reading to indicate actually they are pointed to the same image by default.

Reply:
yeah, added it back.